cli/src/commands/schema/push.rs

```diff
@@ -0,0 +1,13 @@
+use structopt::StructOpt;
+
+use crate::commands::Command;
+use crate::errors::{ExitCode, Fallible};
+
+#[derive(StructOpt)]
+pub struct Push {}
+
+impl Command for Push {
```

Comment:
is the `Command` trait actually doing anything at this point? i introduced it so we wouldn't have to directly reference the implementation files from the command schema, but we're doing that now, so we may as well get rid of the trait